src/main/java/org/sciborgs1155/robot/pivot/PivotConstants.java

```diff
@@ -0,0 +1,71 @@
+package org.sciborgs1155.robot.pivot;
+
+import static edu.wpi.first.units.Units.*;
+import static org.sciborgs1155.robot.pivot.PivotConstants.STARTING_ANGLE;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.Current;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.Mass;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Mult;
+import edu.wpi.first.units.Velocity;
+
+public class PivotConstants {
+  public static final double GEARING = 12.0 / 64.0 * 20.0 / 70.0 * 36.0 / 56.0 * 16.0 / 54.0;
+
+  public static final Measure<Distance> RADIUS = Meters.of(1);
+  public static final Measure<Distance> CIRCUMFERENCE = Meters.of(2 * Math.PI * RADIUS.in(Meters));
+
+  // Offset from the center of the robot to the pivot's axis of rotation
+  public static final Translation2d OFFSET = new Translation2d(Inches.of(10.465), Inches.of(25));
+
+  public static final Measure<Mult<Mult<Distance, Distance>, Mass>> MOI =
+      (Meters).mult(Meters).mult(Kilograms).of(0.17845);
+
+  public static final Measure<Angle> POSITION_FACTOR =
+      Rotations.of(GEARING).times(CIRCUMFERENCE.in(Meters));
+  public static final Measure<Velocity<Angle>> VELOCITY_FACTOR = POSITION_FACTOR.per(Minute);
+
+  public static final Measure<Angle> POSITION_TOLERANCE = Radians.of(0.1);
+
+  public static final Measure<Mass> MASS = Kilograms.of(1);
+  public static final Measure<Distance> LENGTH = Inches.of(16);
+
+  public static final Measure<Velocity<Angle>> MAX_VELOCITY = RadiansPerSecond.of(0.5);
+  public static final Measure<Velocity<Velocity<Angle>>> MAX_ACCEL = MAX_VELOCITY.per(Second);
+
+  public static final Rotation2d MAX_ANGLE = Rotation2d.fromRadians(0);
```

Comment:
switch these names, since the max angle is the starting angle and the arm's angle decreases from there
also make the min angle something negative instead of 0